### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by skumarqa2010**

### CODE REVIEW

File: cypress/integration/teacher-training-adviser.spec.js [hunk 1]
```diff
@@ -134,20 +131,20 @@ describe("Feature - Get an adviser : Tests execution date and time : " + new Dat
...
-		cy.returningToTeaching((returner = false));
+		cy.returningToTeaching(false);
 		cy.doYouHaveDegree("Yes");
 		cy.selectWhatSubjectIsYourDegree("Dance");
 		cy.selectWhichClassIsYourDegree("2:2");
 		cy.wait(1000);
 		cy.selectStage("Primary");
 		cy.gcseMathsAndEnglish(true);
-		cy.doYouHaveGrade4CorAboveInGCSEScienceorEquivalent("Yes");
+		cy.doYouHaveGrade4CorAboveInGCSEScienceorEquivalent(true);
```
Comment (ethax-ross) [inline cypress/integration/teacher-training-adviser.spec.js:147]: I'd be tempted to simplify this to match the other method, so `cy.gcseScience(true)`

**Review by ethax-ross — COMMENTED** [review]
Looks good, just a couple of rename suggestions. It would also be good to reduce the tab spacing a bit, it looks like it's about 6 spaces worth of tabbing; can we reduce it to 2 or 4 (can be one for another PR)?

File: cypress/integration/teacher-training-adviser.spec.js (hunk 1 above)
Comment (skumarqa2010) [reply cypress/integration/teacher-training-adviser.spec.js]: Resolved

File: cypress/integration/teacher-training-adviser.spec.js [hunk 2]
```diff
@@ -211,21 +208,21 @@ describe("Feature - Get an adviser : Tests execution date and time : " + new Dat
...
 		cy.doYouHaveDegree("Yes");
 		cy.selectWhatSubjectIsYourDegree("Dance");
 		cy.selectWhichClassIsYourDegree("2:2");
 		cy.wait(1000);
 		cy.selectStage("Primary");
 		cy.gcseMathsAndEnglish(true);
-		cy.doYouHaveGrade4CorAboveInGCSEScienceorEquivalent("No");
-		cy.areYouPlanningToRetakeYourScienceGCSE("Yes");
+		cy.doYouHaveGrade4CorAboveInGCSEScienceorEquivalent(false);
+		cy.areYouPlanningToRetakeYourScienceGCSE(true);
```
[the remark this answers is not in the document]
Comment (skumarqa2010) [reply cypress/integration/teacher-training-adviser.spec.js]: Resolved

**Review by ethax-ross — APPROVED** [review]
LGTM

**Pull request merged**